newsroom/push.py

```diff
@@ -993,9 +989,7 @@ async def send_topic_notification_emails(item, topics, topic_matches, users, com
             if not user.get("receive_email"):
                 continue
             elif subscriber.get("notification_type") == "scheduled":
-                superdesk.get_resource_service("notification_queue").add_item_to_queue(
-                    user["_id"], section, topic["_id"], item
-                )
+                await NotificationQueueService().add_item_to_queue(user["_id"], section, topic["_id"], item)
```

Comment:
As this line is in nested for loops (over topics and subscribers), it might be good to get the service instance before entering these loops so it's constructor isn't called more than once unnecessarily.